packages/eventual-send/test/test-eventual-send.js

```diff
@@ -1,4 +1,4 @@
-/* global setTimeout */
+/* global setTimeout, window */
```

Comment:
Ah, thanks. I missed that this was behind a `if (typeof window !== 'undefined') `.

I agree it shouldn't be a global. Though I wonder if we should drop the `no-undef` eslint and rely on the type checker to alert on undefined since it can infer from these sorts of type guards.

Reply:
Maybe... it seems somewhat fragile.

Also... dropping `no-undef` would diverge from airbnb community style, yes?

Reply:
I would only be willing to drop the `no-undef` if we can properly tame the global types through a judicious choice of `lib`.

Reply:
I like the explicit comments at the top of the file for all free variables in the file outside of our normal whitelist of powerless globals. That way, I can look at the top of the file and know a bound of the ambient authority that this module accesses by global variable name.
```js
if (typeof window !== 'undefined')
```
is fine as well, but shouldn't bypass the need to declare it as a global. By itself, this if statement doesn't scream ***AMBIENT AUTHORITY*** as it should.